src/app/pages/details/common/get-this-title-files/options.tsx

```diff
@@ -254,8 +254,8 @@ export function KindleOption({model}: {model: Model}) {
 
 export function CheggOption({model}: {model: Model}) {
     return (
-        <Option condition={model.cheggLink}>
-            <a href={model.cheggLink} data-track="Chegg Reader">
+        <Option condition={model.cheggLink ?? false}>
+            <a href={model.cheggLink as string} data-track="Chegg Reader">
```

Comment:
The type for this property seems confusing - it could be null, but not here? Do we need an `assertString` helper?